torch/fx/subgraph_rewriter.py

```diff
@@ -376,7 +378,21 @@ def _replace_pattern(
                     first_user_node = n
                     break
 
-        with original_graph.inserting_before(first_user_node):  # type: ignore[possibly-undefined]
+        first_next_node = None
+        if first_user_node is None:
+            # no users, so we insert the replacement graph before the first next
+            # node of returning nodes
+            next_node = None
+            for n in reversed(original_graph.nodes):
+                if n in match.returning_nodes:
+                    first_next_node = next_node
+                    break
+                else:
+                    next_node = n
+        insert_point = (
+            first_user_node if first_user_node is not None else first_next_node
+        )
+        with original_graph.inserting_before(insert_point):  # type: ignore[possibly-undefined]
```

Comment:
you can fix the linter error by adding something like `assert insert_point is not None`.

Reply:
Thanks for review. I added the assertion and removed the ignore